src/components/Account-panel/Change-password/Change-password.tsx

```diff
@@ -1,3 +1,141 @@
+import React, { useState } from 'react'
+import { useForm } from 'react-hook-form'
+import { yupResolver } from '@hookform/resolvers/yup'
+import axiosInstance from '../../../axios/settings'
+import {
+  Box,
+  Button,
+  FormHelperText,
+  InputLabel,
+  OutlinedInput,
+  Typography
+} from '@mui/material'
+import { CustomSnackbar } from '../../SnackebarCustom/SnackbarCustom'
+import { ChangePasswordSchema } from '../../auth/validation'
+import { SnackbarData } from '../Contact-info/Contact-info'
+
+interface FormData {
+  old_password: string
+  new_password: string
+  new_password_confirm: string
+}
+
+const stInput = {
+  '& input': {
+    backgroundColor: '#ffffff'
+  }
+}
+
+const stBtn = {
+  'backgroundColor': '#FCC812',
+  'color': '#FFFFFF',
+  'marginTop': 4,
+  '&:disabled': {
+    opacity: 0.6,
+    backgroundColor: '#E8E8E8'
+  }
+}
 export const ChangePassword = () => {
-  return <div>ChangePassword</div>
+  const [isLoadingBtn, setIsLoadingBtn] = useState(false)
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    reset
+  } = useForm<FormData>({
+    resolver: yupResolver(ChangePasswordSchema)
+  })
+  const [snackbarData, setSnackbarData] = useState<SnackbarData>({
+    open: false,
+    error: false
+  })
+  const onSubmit = async (data: object) => {
```

Comment:
do not use just `object` as a type, clarify what object and which keys it has

Reply:
done